reports/AC187/report_for_AC187_cleanup.rb

```diff
@@ -0,0 +1,60 @@
+require 'archivesspace/client'
+require 'json'
+require 'csv'
+require_relative '../../helper_methods.rb'
+
+aspace_staging_login()
+
+start_time = "Process started: #{Time.now}"
+puts start_time
+
+eadid = "AC187"
+resource_id = "4185"
+output_file = "#{eadid}.csv"
```

Comment:
@regineheberlein it would be good to put the data into the data directory, instead of local.  Depending on if you are planning to run this locally, or just on the server, you could use a relative path like '../../data' or a hard coded path like '/opt/aspace_helpers/current/data'

Reply:
Yes! Right now, running these locally, I have a project-based directory structure that allows me to easily access everything pertaining to any given project. I'll rethink this on the the server for sure.